core/src/test/scala/unit/kafka/server/AbstractApiVersionsRequestTest.scala

```diff
@@ -24,10 +24,13 @@ import org.apache.kafka.common.protocol.ApiKeys
 import org.apache.kafka.common.requests.{ApiVersionsRequest, ApiVersionsResponse, RequestUtils}
 import org.apache.kafka.common.utils.Utils
 import org.junit.jupiter.api.Assertions._
-
 import java.util.Properties
+
+import org.junit.jupiter.api.Tag
```

Comment:
Why is this jupiter include in its own section, and not with the other jupiter includes?